num2words/lang_ES.py

```diff
@@ -176,4 +349,10 @@ def to_currency(self, val, currency='EUR', cents=True, separator=' con',
             val, currency=currency, cents=cents, separator=separator,
             adjective=adjective)
         # Handle exception, in spanish is "un euro" and not "uno euro"
+        # except in this currencies: leona, corona,
+        # libra, rupia, lempira, peseta, is 'una'
+        for currency in CURRENCIES_UNA:
+            if currency in result:
+                result = result.replace("uno", "una")
+                break
```

Comment:
```suggestion
        # except in this currencies: leona, corona,
        # libra, rupia, lempira, peseta, is 'una'
        return result.replace("uno", "una") if currency in CURRENCIES_UNA else result.replace("uno", "un")
```

Reply:
@erozqba 

I updated that piece of code, to replace "uno" by "una" is only in case singular, because in other cases, for example in 'veintiuno libras con cero centavos', is not neccesary change 'veintiuno' by 'veintiuna'.

Regards.